src/routes/(app)/playlist/zoek/Tracklist.svelte

```diff
@@ -0,0 +1,214 @@
+		} else {
+			liked = [...liked, track.id];
+			isLiked = true;
+		}
+
+		try {
+			await fetch('/playlist', {
+				method: 'POST',
+				body: JSON.stringify({
+					trackId: track.id,
+					liked: isLiked
+				})
+			});
+		} catch (error: any) {
+			toast({
+				title: 'Error',
+				message: error,
+				type: 'error'
+			});
+		}
+	};
+</script>
+
+<audio src={previewSrc} bind:this={audioPlayer} />
+<ul>
+	{#each tracks as track}
+		<li class={playlist.includes(track.id) ? 'highlight' : ''}>
+			<!-- svelte-ignore a11y-click-events-have-key-events -->
+			<!-- svelte-ignore a11y-no-static-element-interactions -->
+			<div
+				on:click={async () => {
+					await toggleLike(track);
+				}}
+				class="like"
+			>
+				{#if liked.includes(track.id)}
+					<HeartFilled color="#1db954" />
+				{:else}
+					<Heart />
+				{/if}
+			</div>
+			<!-- svelte-ignore a11y-click-events-have-key-events -->
+			<!-- svelte-ignore a11y-no-noninteractive-element-interactions -->
+			<!-- svelte-ignore a11y-no-static-element-interactions -->
+			<div
+				on:click={() => {
+					hovering = 'een trigger zodat svelte het weer snapt';
+					hovering = track.preview_url ?? '';
+					if (track.preview_url === previewSrc) {
+						if (audioPlayer.paused) audioPlayer.play();
+						else audioPlayer.pause();
+					} else {
+						if (track.preview_url) previewSrc = track.preview_url;
+					}
+				}}
+				class={`${track.preview_url ? 'clickable' : ''}`}
+				on:mouseenter={() => (hovering = track.preview_url ?? '')}
+				on:mouseleave={() => (hovering = '')}
+			>
+				<img
+					src={getSmallestImage(track.album.images).url}
+					alt={'Album cover ' + track.name}
+					class={`${previewSrc === track.preview_url && !audioPlayer.paused ? 'highlight' : ''}`}
+				/>
+				{#if track.preview_url}
+					{#if (hovering === track.preview_url && (previewSrc !== track.preview_url || audioPlayer.paused)) || (previewSrc === track.preview_url && audioPlayer.paused)}
+						<Play style="position: absolute; top: 16px; left: 16px; height: 26px; width: 26px;" />
+					{:else if previewSrc === track.preview_url && !audioPlayer.paused}
+						<Pause style="position: absolute; top: 16px; left: 16px; height: 26px; width: 26px;" />
+					{/if}
+				{/if}
+			</div>
+			<div class="info">
+				<a class="title" href={track.external_urls.spotify} target="_blank">{track.name}</a>
+				<p class="artists">{formatArtists(track.artists)}</p>
+			</div>
+		</li>
+	{/each}
+</ul>
+
+<style lang="scss">
+	$highlight-opacity: 0.7;
+
+	ul {
+		li {
+			display: grid;
+			grid-template-columns: 20px 58px 1fr;
+			grid-template-rows: 58px;
+			align-items: center;
+			gap: 20px;
+
+			&.highlight {
+				$hightlight-color: var(--primary-light-color);
+
+				background-color: $hightlight-color;
+				box-shadow: $hightlight-color -20px 0px 0px 5px, $hightlight-color 20px 0px 0px 5px;
+
+				a,
+				p,
+				* {
+					color: white !important;
+				}
+			}
+
+			.like {
+				$user-select: none;
```

Comment:
Waarom is deze variabele none? Overschijf je deze nog ergens dynamisch ofzo? Dat is iig niet mogelijk met scss, aangezien alle variabelen vervangen worden door deze waarde op compile time